project/kevinshlo/README.md

```diff
@@ -0,0 +1,86 @@
+# Corner Stitching
+
+312551134 羅時弘
+
+## Basic Information
+
+"Corner Stitching" is a fundamental data structure for tile-based detailed routing in EDA.  
+The code of this project is hosted on GitHub [kevinshlo/CornerStitching](https://github.com/kevinshlo/CornerStitching) .
+
+## Problem to Solve
+
+In electronic design automation (EDA), the last step of circuit design is detailed routing, which is considered the most difficult and time consuming step. There are several categories of methods to address this problem, and one of them is called tile-based detailed routing. In which, corner stiching is a fundamental data structure to represent tiles on routing layers.
+
+Recently, routers involves deep learning have gained a lot of research attention. There is a need to make accessing routing data structures from DL tools such as PyTorch more efficient. This project aims to provide a Python interface for accessing corner stiching data structure in C++ efficiently.
+
+## Prospective Users
+
+Researchers in EDA field who want to introduce DL into tile-based routers might be interested with this tool.
+
+## System Architecture
+
+This tool will be developed as a C++ library and providing a Python interface. The corner stiching data structures holding the tiles is managed by C++ library, while users can call APIs to access and modifies tiles.
+
+A class `RoutingPlane` contains several `Layer` and other meta information about the whole routing plane.
```

Comment:
It may be clearer to write down the header with code comments.  When it comes to data structure design, programming language works well.